packages/features/ee/teams/components/MemberInvitationModal.tsx

```diff
@@ -94,7 +94,14 @@ export default function MemberInvitationModal(props: MemberInvitationModalProps)
 
     const inviteLink =
       isOrgInvite || (props?.orgMembers && props.orgMembers?.length > 0) ? orgInviteLink : teamInviteLink;
-    await navigator.clipboard.writeText(inviteLink);
+
+    try {
+      await navigator.clipboard.writeText(inviteLink);
+      console.log(inviteLink);
+    } catch (e) {
+      console.log(inviteLink);
+    }
```

Comment:
Added try/catch to avoid the error we get when using a baseURL that is not localhost such as app.cal.local, and in either case logging the invite link to the console to be able to use it in tests

Reply:
We should make this explicit. By reading this alone, no one can infer that removing these console logs will cause tests to fail.